sim/midas/src/main/scala/midas/stage/Annotations.scala

```diff
@@ -45,3 +48,16 @@ object RuntimeConfigNameAnnotation extends HasShellOptions {
       shortOption = Some("ggrc"),
       helpValueName = Some("<filename>") ) )
 }
+
+case class OutputBaseFileNameAnnotation(name: String) extends NoTargetAnnotation with GoldenGateOption
+
+object OutputBaseFileNameAnnotation extends HasShellOptions {
```

Comment:
See, you put Output first in this class name.  I think `OutputFileNameBaseAnnotation` would still be a clearer name.  Related to my first comment on the docs and CLI option name.

Reply:
Maybe `OutputBaseFileNameAnnotation` is better because it `OutputFileNameBaseAnnotation` seems to be implying something about a `BaseAnnotation` and that isn't a thing (or maybe it is but it isn't something we're using here [at least not directly] )

Reply:
I would make the CLI argument consistent with the name here  so either `--output-base-file-name` or `--output-file-name-base`

Reply:
You should tell me what you think about the RC approach.  Otherwise, i'll switch uniformly to output-base-file-name.

Reply:
Switch to `output-base-file-name`.  If GH had the vomit emoji, I'd use it to express how I feel about the RC option name. 🤮   Oh, it does have the 🤮 emoji!  yay.  That's how I feel about it.